src/core/utils/cli-config.ts

```diff
@@ -19,9 +19,9 @@ export const getCurrentSwaCliConfigFromFile = () => currentSwaCliConfigFromFile;
 export const swaCliConfigFileExists = (configFilePath: string) => existsSync(configFilePath);
 
 export async function getConfigFileOptions(
-  contextOrConfigEntry: string | undefined,
+  outputLocationOrConfigEntry: string | undefined,
```

Comment:
I think we should keep context internal in this file, as using outputLocation instead makes it more confusing for commands that don't include one (ie login/init but also upcoming build link etc).

Reply:
Why rename `contextOrConfigEntry` to `outputLocationOrConfigEntry`?

Currently, this is how `getConfigFileOptions(a, b)` can be invoked:
- `swa login` : `a` is undefined => we skip loading any config
- `swa init name`:  `a` is undefined => we skip loading any config
- `swa start ./folder`:  context is outputLocation
- `swa start http://localhost:3000`: context is outputLocation (aka devServerUrl)
- `swa start name`: context is configEntry
- `swa deploy ./folder`:  context is outputLocation
- `swa deploy name`:  context is configEntry